neural_compressor/adaptor/torch_utils/gptq.py

```diff
@@ -61,6 +76,9 @@ def trace_gptq_target_blocks(module, module_types = [torch.nn.ModuleList]):
     return gptq_related_blocks
 
 def find_layers(module, layers=[nn.Conv2d, nn.Linear], name=''):
+    """

```

Comment:
does transformer.conv1d need to be supported here?

Reply:
Added. module type of transformers.Conv1D now can be found in models like LaMini-GPT-xxx. I also support torch.nn.Conv1d, whose weights flatten process is similar to torch.nn.Conv2d, I will find a model containing torch.nn.Conv1d to have a try.